styles.css

```
@@ -144,6 +144,42 @@ body {
     color: inherit !important;
   }
+
+  /* Prevent content from overflowing off the printed page */
+  * {
+    max-width: 100%;
+    box-sizing: border-box;
+  }
+
+  pre, code {
+    white-space: pre-wrap;
+    word-break: break-word;
```

Stylelint (17.3.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)